### ISSUE TITLE
Ability to merge Custom Field definitions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#175]
```
What functionality do you want to see?

If you've got "books" and "books2" as 2 defined custom fields, there should be 
a way to merge them.  One definition will have to win out, but this will update 
the "meta_key" in the database to the target value.

E.g. "Merge this field into field ________" will translate to 

UPDATE wp_postmeta SET meta_key='target_value' WHERE meta_key='old_value';


```

Original issue reported on code.google.com by `fireproofsocks` on 12 Sep 2011 at 5:16

**Comment by GoogleCodeExporter** [comment]
```
Ok, this has been implemented.
Committed revision 440590.

Will be available in version 0.9.4
```

Original comment by `fireproofsocks` on 20 Sep 2011 at 4:27

**Comment by GoogleCodeExporter** [comment]
Original comment by `fireproofsocks` on 20 Sep 2011 at 4:27
- Changed state: **Fixed**
